_posts/2025-08-19-cockpit-345.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 345
+author: mvo
+date: '2025-08-19'
+tags: cockpit, machines
+slug: cockpit-345
+category: release
+summary: 'IPv6 for Wireguard, detachable VNC console'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 345 and cockpit-machines 338:
+
+
+## Networking: Support IPv6 addresses for Wireguard
+
+The Networking page can now configure IPv6 addresses for WireGuard VPN
+directly within the WireGuard modal.
+
+<img width="840" height="684" alt="Screen Shot 2025-08-20 at 14 16 46" src="https://github.com/user-attachments/assets/9453ee37-12b8-4882-b8dd-b638cdc1b606" />
+
+## Machines: The VNC console of a virtual machine can be detached
```

Comment:
This is a full sentence and the word "detached" is confusing.  Like the first thought I have is that you detach the connection or the serial device or something.  I'd rewrite it something like "Allow pop-out of VM console".